### DISCUSSION TITLE
Problem with Digit ROI

### DISCUSSION THREAD

**Discussion by fudi6489** [jomjol/AI-on-the-edge-device#3011] [Q&A] [answered]
Hello

I'm having a problem detecting my Digit ROI. The analog ones always work perfectly.

I have already tried all model versions in the config but there was no improvement.
I have already adjusted the DigitROI and managed to position the number nicely in the inner frame. According to the instructions, the outer frame should be set to the upper and lower edge of the number window. Unfortunately, this is not possible for me because I cannot adjust the size of the outer frame.
Unfortunately, my numbers are also clearly in the lower area between the two black frames, which could exacerbate the problem.

I would be very grateful for tips. I don't think the quality of the image is bad, but since it recognizes an 8 instead of a 3, there must be something wrong in the settings.

![Zähler 1](https://github.com/jomjol/AI-on-the-edge-device/assets/128504806/5a2cfdec-0a77-4b7b-9613-d8409e3cbf08)
![Zähler 2](https://github.com/jomjol/AI-on-the-edge-device/assets/128504806/506088e5-a0dd-4167-9e5e-9428dbb74d79)
![Zähler 3](https://github.com/jomjol/AI-on-the-edge-device/assets/128504806/59e2c7d7-d108-433e-8c9b-7e42066cfbdd)

Greetings from Austria
Jurgen

**Comment by SybexX** [comment]
![Unbenannt](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/73f0817e-4b18-46ba-ace4-8921334197ef)

**Reply by fudi6489** [reply]
Thanks for the reply

I've already done that. But I can't change the outer frame and it results from the inner frame. Since my numbers are not in the middle, the outer frame is clearly in the black area when the inner frame is positioned correctly.

**Reply by SybexX** [reply, marked as answer]
It looks like there is a reflection of the light at the bottom and a shadow at the top, so I would try turning the camera 180°.

**Reply by fudi6489** [reply]
@SybexX 
i have turned the camera 180°, and since two days it works nice. Hope it stays that way. Thanks for your help.
